SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+            
+            Circle()
+                .trim(from: 0, to: climateControlViewModel.isClimateControlEnabled ? climateControlViewModel.climateSliderValues[climateControlViewModel.settingsIndex] : 0)
+                .stroke(LinearGradient(colors: [Color(climateControlViewModel.bottomGradientColor), Color(climateControlViewModel.topGradientColor)], startPoint: .top, endPoint: .bottomTrailing), lineWidth: 15)
+                .frame(width: 168)
+                .rotationEffect(.degrees(-90))
+                .shadow(color: Color(climateControlViewModel.bottomGradientColor).opacity(0.5), radius: 3, x: -3, y: -3)
+                .shadow(color: Color(climateControlViewModel.topGradientColor).opacity(0.5), radius: 3, x: 3, y: 3)
+        }
+    }
+    
+    private var gradient: LinearGradient {
+        LinearGradient(colors: [Color(Constants.gradientTopColorName), Color(Constants.gradientBottomColorName)], startPoint: .bottom, endPoint: .top)
+    }
+    
+    private var bottomSheetView: some View {
+        VStack {
+            ZStack {
+                RoundedRectangle(cornerRadius: 30)
+                    .fill(gradient).opacity(0.15)
+                
+                ZStack {
+                    HStack {
+                        VStack(spacing: -10) {
+                            HStack {
+                                Text("A/C is ON")
+                                    .font(Font.system(size: 18))
+                                    .bold()
+                                Spacer()
+                            }
+                            HStack {
+                                Text("Tap to turn off or swipe up for a fast setup")
+                                    .font(Font.system(size: 15, weight: .semibold))
+                                    .foregroundColor(.gray)
+                                Spacer()
+                                СonvexCircleButtonView(actionHandler: {
+                                    climateControlViewModel.isClimateControlEnabled.toggle()
+                                }, iconName: "on")
+                            }
+                        }
+                       
+                        Spacer()
+                    }
+                }
+                .frame(width: UIScreen.main.bounds.width - 20)
+                
+                
+                VStack {
+                    Capsule()
+                        .fill(.black)
+                        .frame(width: 70, height: 5)
+                        .padding(.top)
+                    Spacer()
+                }
+            }
+            .frame(height: 100)
+                
+            VStack {
+                ZStack {
+                    RoundedRectangle(cornerRadius: 30)
+                        .fill(gradient).opacity(0.15)
+                    
+                    
+                    ZStack {
```

Comment:
```suggestion
                    ZStack {
```
разбей на маленькие вью